### ISSUE TITLE
Small question.

### ISSUE THREAD

**Issue by BahaaMansour** [AlexeyAB/darknet#2211]
During training, avg loss is not getting lower than 4.0 @ 0.001 learning rate.
Does that mean i have to lower the learning rate to 0.0001? what will happen if i do that?

@AlexeyAB 

Thank you

**Comment by AlexeyAB** [comment]
@BahaaMansour 

> During training, avg loss is not getting lower than 4.0 @ 0.001 learning rate.

May be your dataset is difficult.

> Does that mean i have to lower the learning rate to 0.0001? what will happen if i do that?

Just try to decrease learning_rate and continue train. It can help.
